owid/datautils/geo.py

```diff
@@ -1,3 +1,491 @@
 """Utils related to geographical entities.
 
 """
+
+import json
+import warnings
+from typing import List, Union, Set, Optional, Dict, Any
+
+import numpy as np
+import pandas as pd
+import owid.catalog as catalog
+
+from owid.datautils.common import ExceptionFromDocstring
+from owid.datautils.dataframes import groupby_agg
+
+# When creating region aggregates for a certain variable in a certain year, some mandatory countries must be
+# informed, otherwise the aggregate will be nan (since we consider that there is not enough information).
+# A country will be considered mandatory if they exceed this minimum fraction of the total population of the region.
+MIN_FRAC_INDIVIDUAL_POPULATION = 0.0
+# A country will be considered mandatory if the sum of the population of all countries (sorted by decreasing
+# population until reaching this country) exceeds the following fraction of the total population of the region.
+MIN_FRAC_CUMULATIVE_POPULATION = 0.7
+# Reference year to build the list of mandatory countries.
+REFERENCE_YEAR = 2018
+# Maximum fraction of nans allowed per year when doing aggregations (None to allow any fraction of nans).
+FRAC_ALLOWED_NANS_PER_YEAR = 0.2
+# Maximum number of nans allowed per year when doing aggregations (None to allow any number of nans).
+NUM_ALLOWED_NANS_PER_YEAR = None
+
+
+def _load_population() -> pd.DataFrame:
+    population = (
+        catalog.find("population", namespace="owid", dataset="key_indicators")
+        .load()
+        .reset_index()
+    )  # type: pd.DataFrame
```

Comment:
Nitpick - AFAIK type annotations should be preferred over type comments. You can write this as
```
population: pd.DataFrame = (
        catalog.find("population", namespace="owid", dataset="key_indicators")
        .load()
        .reset_index()
    )
```
or 
```
return cast(pd.DataFrame, population)
```

Reply:
Thanks, done.